lua/user/telescope.lua

```diff
@@ -169,6 +169,19 @@ telescope.setup {
     colorscheme = {
       -- enable_preview = true,
     },
+    lsp_references = {
+      theme = "dropdown",
+    },
+    lsp_definitions = {
+      theme = "dropdown",
+    },
+    lsp_declarations = {
+      theme = "dropdown",
+    },
+    lsp_implementations = {
+      theme = "dropdown",
+    },
```

Comment:
Note to self: I am not sure how much I like this theme of `dropdown`. But I'm going to try it, and let's iterate again if it's not good.